diff --git a/qtgui/uiprefs.ui b/qtgui/uiprefs.ui
index 21c22180..2327ac11 100644
--- a/qtgui/uiprefs.ui
+++ b/qtgui/uiprefs.ui
@@ -718,7 +718,7 @@
                 </size>
                </property>
                <property name="editable">
-                <bool>no</bool>
+                <bool>false</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::NoInsert</enum>
